services/submission.js

```diff
@@ -0,0 +1,129 @@
+const { NEW, IN_PROGRESS, SUBMITTED, RELEASED, COMPLETED, ARCHIVED} = require("../constants/submission-constants");
+const {v4} = require('uuid')
+const {getCurrentTime} = require("../crdc-datahub-database-drivers/utility/time-utility");
+const {HistoryEventBuilder} = require("../domain/history-event");
+const {verifySession} = require("../verifier/user-info-verifier");
+const {getSortDirection} = require("../crdc-datahub-database-drivers/utility/mongodb-utility");
+const {formatName} = require("../utility/format-name");
+const ERROR = require("../constants/error-constants");
+const USER_CONSTANTS = require("../crdc-datahub-database-drivers/constants/user-constants");
+const ROLES = USER_CONSTANTS.USER.ROLES;
+const ALL_FILTER = "All";
+
+
+
+function listConditions(userID, userRole, aUserOrganization, params){
+    const validApplicationStatus = {status: {$in: [NEW, IN_PROGRESS, SUBMITTED, RELEASED, COMPLETED, ARCHIVED]}};
+    // Default conditons are:
+    // Make sure application has valid status
+    let conditions = {...validApplicationStatus};
+    // Filter on organization and status
+    if (params.organization !== ALL_FILTER) {
+        conditions = {...conditions, "organization.name": params.organization};
```

Comment:
If the organization does not exits, do you think we should throw an error?

Reply:
Frontend will only provide organizations from the mongoDB database. So i don't think it is possible for the organization to not exist